src/demo/apps/DialerApp.tsx

```diff
@@ -1,114 +1,91 @@
 import * as React from 'react';
 import * as fdc3 from '../../client/index';
+import {Number} from '../components/dialer/Number';
+import {Dialer} from '../components/dialer/Dialer';
+import {CallTimer} from '../components/dialer/CallTimer';
+import {CallButton} from '../components/dialer/CallButton';
+import {ContactPayload, Payload} from '../../client/context';
+import {Dialog} from '../components/common/Dialog';
 
 import '../../../res/demo/css/w3.css';
+import {ColorLinker} from '../components/common/ColorLinker';
 
-import { Number } from '../components/dialer/Number';
-import { Dialer } from '../components/dialer/Dialer';
-import { CallTimer } from '../components/dialer/CallTimer';
-import { CallButton } from '../components/dialer/CallButton';
-import { ContactPayload, Payload } from '../../client/context';
-import { Dialog } from '../components/common/Dialog';
-
-interface IAppProps {
+interface AppProps {
     phoneNumber?: string;
 }
 
-interface IAppState {
-    inCall: boolean;
-    phoneNumber: string;
-    pendingCall: ContactPayload|null;
-}
+// tslint:disable-next-line:variable-name
```

Comment:
Are we really going to need a ts-lint exception before every component declaration in the whole service?

Reply:
Can add the following to the `tslint.json`?:
```
"variable-name": {
            "options": [
                "allow-pascal-case"
            ]
        },
```

But that will allow pascal case everywhere.

Reply:
Can we make these declarations functions, rather than constants? As in `export function DialerApp(props: AppProps) {`?

Firstly, because... well, they are actually functions (rather than variables/constants). But should also remove the need for the linter exception?..

Reply:
It's the "react" way to do fat arrow functions for FC.  However, as Phil noted it, making them old school functions will fix the linting errors as we have seen before.  We should probably go this route to avoid sprinkling tslint disables everywhere. 

Something to note is that you will lose the `React.FunctionComponent<...>` type.  You will need to assign a type directly to `props` and if you use children, it'll be `props: React.PropsWithChildren<PropType>`.  Maybe more will be needed to get `useEffect` to behave.

Reply:
Looking at the definition of `FunctionComponent`, it's a function that returns a `ReactElememt`.

Should be strongly-typable as so:
```
export function DialerApp(props: AppProps): React.ReactElement<AppProps> {
    // . . .
}
```